From 444634d552f5a9765ee46e2dfa08180ecafa3f21 Mon Sep 17 00:00:00 2001
Message-Id: <444634d552f5a9765ee46e2dfa08180ecafa3f21.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Jon Bloomfield <jon.bloomfield@intel.com>
Date: Mon, 7 Jul 2014 16:06:00 +0100
Subject: [PATCH 101/312] REVERTME [VPG]: Disable flipwq

This patch can be reverted when
	"FOR_UPSTREAM [VPG]: drm/i915/vlv: Replaced Blitter ring based flips with MMIO Flips, in pageflip call."
is rebased to the forklift

Change-Id: I380d04e078c6fb1bb19632de750e13d1100b5ab5
---
 drivers/gpu/drm/i915/intel_display.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index eb6b267..b3d477a 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -3864,7 +3864,9 @@ void intel_crtc_wait_for_pending_flips(struct drm_crtc *crtc)
 	flush_work(&to_intel_crtc(crtc)->vblank_work.work);
 
 	WARN_ON(waitqueue_active(&dev_priv->pending_flip_queue));
+#if 0
 	flush_workqueue(dev_priv->flipwq);
+#endif
 
 	obj = to_intel_framebuffer(crtc->primary->fb)->obj;
 	if (wait_event_timeout(dev_priv->pending_flip_queue,
-- 
1.7.9.5

